shell/src/main/java/alluxio/cli/fsadmin/command/BackupCommand.java

```diff
@@ -63,12 +65,21 @@ public int run(CommandLine cl) throws IOException {
       opts.setTargetDirectory(args[0]);
     }
     opts.setLocalFileSystem(cl.hasOption(LOCAL_OPTION.getLongOpt()));
-    BackupResponse resp = mMetaClient.backup(opts.build());
+    // Create progress thread for showing progress while backup is in progress.
```

Comment:
Currently the detailed backup progress is shown in leader master.log, looks like we are not able to print the detailed backup progress to users directly. Can we add some user-facing console message to tell them the detailed backup progress is in leader master master.log?

Reply:
We can, but details we can provide is not very interesting for a shell user. We cannot show percentage or estimation. We can only show how many entries applied so far, which is not indicative of anything, that's why we decided to go with simple dot-dot progress.